packages/xsnap/test/test-xs-js.js

```diff
@@ -50,6 +51,61 @@ test('simple TextEncoder / TextDecoder are available', async t => {
   t.deepEqual(opts.messages, ['"Hello! 😊"', '"A\\u0000A"']);
 });
 
+test('Base64.encode', async t => {
```

Comment:
for the other FFI bindings, we test that they are _only available in the start compartment_.

When adding Base64, we should probably refactor those tests to just handle a list of things.